src/quo2/components/info/page_nav.cljs

```diff
@@ -0,0 +1,242 @@
+(ns quo2.components.info.page-nav
+  (:require [quo.react-native :as rn]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.icon :as icons]
+            [status-im.utils.dimensions :as dimensions]
+            [clojure.string :as string]
+            [quo2.components.markdown.text :as text]))
+
+(def ^:private centrify
+  {:style {:display :flex
```

Comment:
if it has only style probably its better to name `centrify-style` in that case you won't need update-in and can just directly assoc, will be much better